maap/config_reader.py

```diff
@@ -81,6 +81,7 @@ def __init__(self, maap_host):
         self.algorithm_build = self._get_api_endpoint("algorithm_build")
         self.mas_algo = self._get_api_endpoint("mas_algo")
         self.dps_job = self._get_api_endpoint("dps_job")
+        self.member_secrets = self._get_api_endpoint("member_secrets")
```

Comment:
Has this new config also been added to the maap_api_settings file as default config ? here https://github.com/MAAP-Project/maap-api-nasa/blob/main/api/settings.py#L119-L132

Reply:
I went with the existing member endpoint.